#!/usr/bin/env python
PACKAGE = "ros_rrbot_lecture"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("IK_gain", double_t, 0, "Gain of closed-loop inverse kinematics",    1.0, 0.0,   1000.0)
gen.add("pseudoinv_damping", double_t, 0, "Damping factor for damped jacobian pseudo-inverse",    0.1, 0.0,   10.0)

exit(gen.generate(PACKAGE, "ros_rrbot_lecture", "Controller"))